tests/Wallet/TestWallet.nim

```diff
@@ -0,0 +1,65 @@
+import unittest2
+
+#Util lib.
+import ../../src/lib/Util
+
+#Wallet libs.
+import ../../src/Wallet/Address
+import ../../src/Wallet/Wallet
+
+#Random standard lib.
+import random
+
+const TEST_WALLET_ITERATIONS {.intdefine.}: int = 1
+
+proc verify(
+    wallet: Wallet
+) =
+    #Test recreating the Public Key.
+    assert(newEdPublicKey(wallet.publicKey.toString()).toString() ==
+        wallet.publicKey.toString())
+    assert($newEdPublicKey($wallet.publicKey) == $wallet.publicKey)
+
+    #Create messages.
+    var
+        msg: string
+        sig: EdSignature
+    for m in 1 .. rand(100):
+        msg = ""
+        for _ in 0 ..< m:
+            msg &= char(rand(255))
+
+        #Sign the message.
+        sig = wallet.sign(msg)
+
+        #Verify the signature.
+        assert(wallet.verify(msg, sig))
+
+suite "Wallet":
+    setup:
+        randomize(int64(getTime()))
+
+    test "new wallet without password":
+        for _ in 1 .. TEST_WALLET_ITERATIONS:
+            verify(newWallet(""))
+
+    test "new wallet with password":
```

Comment:
Why not use "New Wallet with password."?

Reply:
_Because proper sentences are hard. :P_

Proper sentences seems to be a convention of this project. I'm ok with it, just not used to it is all.

Reply:
I never understood why you wouldn't use them :p That said, I understand my styling has certain intricacies and am willing to find common ground as it benefits the project.